test-cases/longevity/longevity-encryption-at-rest-200GB-6h.yaml

```diff
@@ -0,0 +1,31 @@
+test_duration: 380
```

Comment:
Need more time, the c-s is 360 without the prepare and setup.

Reply:
http://jenkins.scylladb.com/view/enterprise-2019.1/job/enterprise-2019.1/job/longevity-tests/job/longevity-200GB-with-verify-aws-test/17/

This job took 2 days + 3hours + 10 mins, cs workload duration is 2 days.

So let's use 6h + 3h 10mins = 550 mins